tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1473,6 +1474,100 @@ def test_submissions__with_pause_and_skip_after(self, _):
             assert submission_count < 100
 
 
+class TestSubredditModerationStreams(IntegrationTest):
+    def setup_reddit(self):
+        self.reddit = Reddit(
+            client_id=pytest.placeholders.client_id,
+            client_secret=pytest.placeholders.client_secret,
+            refresh_token=pytest.placeholders.refresh_token,
+            redirect_uri=pytest.placeholders.redirect_uri,
+            user_agent=pytest.placeholders.user_agent,
+        )
```

Comment:
This method is not used in the class. Can it be removed?

Reply:
No, it is to enable the use of a refresh token. Instead of checking for a refresh token at the beginning and making a reddit instance with token auth, I thought it would be better to have the tests that use token init the reddit class themselves. Also, I felt that it would be better to test them on an account with a lot of large subs like mine (u/Lil_SpazJoekp). My account has 2FA enabled and I not willing to disable it due to some of the subs I moderate.

Reply:
What I mean is that nowhere else in the file does it seem the method is called and thus it is unreachable code.

Reply:
it is called before test is ran. This was done here as well:
https://github.com/praw-dev/praw/blob/master/tests/integration/models/test_auth.py#L16-L37

Reply:
It is an override of this here: 
https://github.com/praw-dev/praw/blob/2b9e686b76c92c59dbf6c341db48345fd21fbade/tests/integration/__init__.py#L27-L34